doc/source/autoscaling.rst

```diff
@@ -50,6 +52,28 @@ SSH into the head node and then run Ray programs with ``ray.init(redis_address="
     # Teardown the cluster
     $ ray down ray/python/ray/autoscaler/gcp/example-full.yaml
 
+Quick start (Local)
```

Comment:
"Local" might be misleading, maybe say "Local cluster of machines"?

Reply:
How about "Local" -> "Private"? As opposed to a "Public" cloud.

Reply:
I mean this suggestion not just for this line, but everywhere.